scripts/histmakers/w_z_gen_dists.py

```diff
@@ -150,37 +148,37 @@ def build_graph(df, dataset):
 if not args.skipAngularCoeffs:
     for dataset in datasets:
         name = dataset.name
-        if "helicity_moments_scale" not in resultdict[name]:
+        if "helicity_moments_scale" not in resultdict[name]['output']:
             logger.warning(f"Failed to find helicity_moments_scale hist for proc {name}. Skipping!")
             continue
         moments = resultdict[name]["output"]["helicity_moments_scale"]
         if name in common.zprocs:
             if z_moments is None:
-                z_moments = moments
+                z_moments = hh.makeAbsHist(moments.get(), "y")
             else:
-                z_moments += moments
+                new_moments = hh.makeAbsHist(moments.get(), "y")
+                z_moments = hh.addHists(z_moments, new_moments)
         elif name in common.wprocs:
             if w_moments is None:
-                w_moments = moments
+                w_moments = hh.makeAbsHist(moments.get(), "y")
             else:
-                w_moments += moments
-
-if z_moments and w_moments:
+                new_moments = hh.makeAbsHist(moments.get(), "y")
+                w_moments = hh.addHists(w_moments, new_moments)
+    yax_name = "absy"
+    coeffs={}
     # REMINDER: common.ptV_binning is not the one using 10% quantiles, and the quantiles are not a subset of this binning, but apparently it doesn't matter
-    yax_name = axis_absYVgen.name
-    if "y" in z_moments.axes.name and "y" in w_moments.axes.name:
```

Comment:
Would be good to keep this check in case we run with absYVgen

Reply:
Okay will put it back.